internal/static/testdata/apps/no-app/app.go

```diff
@@ -0,0 +1,7 @@
+package app
+
+// App does not implement dogma.Application interface.
+type App struct{}
```

Comment:
```suggestion
type NonApp struct{}
```

As best I can tell this test is about ignoring a type that is NOT an app, so the type could be named that way?